radixdlt-core/radixdlt/src/main/java/com/radixdlt/client/store/berkeley/BerkeleyClientApiStore.java

```diff
@@ -550,6 +542,10 @@ private void rebuildDatabase() {
 		log.info("Database rebuilding is finished successfully");
 	}
 
+	public EventProcessor<AtomsCommittedToLedger> atomsCommittedToLedgerEventProcessorBerkeleyClientApi() {
```

Comment:
this method is already in `BerkeletClientApiStore`, no need to add suffix to the name